src/commands/api/generate.ts

```diff
@@ -0,0 +1,41 @@
+import { spawn } from 'node:child_process';
+
+import { Command } from '@oclif/core';
+
+import logger from 'utils/logger';
+
+export default class ApiGenerate extends Command {
+  DEBUG_TAG = 'api:generate';
+
+  static description = `Prepares your environment for custom queries development.
+                        run it from your root directory and it will create all neccesary files and scripts
+                        to start working with custom api queries and mutations.`;
+
+  async run() {
+    try {
+      await this.runSetupApi();
+      logger.info('setup-api completed successfully.');
+    } catch (error) {
+      logger.debug(error, this.DEBUG_TAG);
+      logger.error(`setup-api failed, please make sure you run it in your root directory and try again}`);
```

Comment:
throw the error OR your error in the end so you would get the correct process.exit code

Reply:
handled by the base classes

Reply:
done